<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntitySelectionWidget</class>
 <widget class="QWidget" name="EntitySelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>172</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entity Selection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/breezEd.qrc">
    <normaloff>:/breezEd/icons/tools/selection</normaloff>:/breezEd/icons/tools/selection</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="selectionList">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QToolButton" name="assetButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>To Asset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="groupButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>To Group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deselectButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Deselect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/breezEd.qrc"/>
 </resources>
 <connections/>
</ui>
